juniper_official/routing/check-bgp-neighbor-stats.rule

```diff
@@ -92,7 +92,7 @@
                     when {
                         increasing-at-least-by-value "$flap-count" {
                             value "$flap-count-threshold";
-                            time-range 3offset;
+                            time-range 2.5offset;
```

Comment:
matches-with-previous should also have 2.5 offset

Reply:
Resolved